inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_mul_add_to_scaleshift_or_power.cpp

```diff
@@ -17,11 +17,11 @@
 #include "ngraph_ops/scaleshift.hpp"
 
 CONVERSION_RESULT check_constant(const std::shared_ptr<ngraph::opset1::Constant>& constant,
-                                 const ngraph::Shape& shape) {
-    if (!constant) return CONVERSION_RESULT::NONE;
+                                 const ngraph::PartialShape& shape) {
```

Comment:
It will be great to have a comment describing what this function does. The same comment for other utility functions that you use here.

Reply:
That's not utility function and they are defined inside transformation. But anyway I'll add comments.

Reply:
Done